README.md

```diff
@@ -147,6 +147,15 @@ CVAT annotations                             ---> Publication, statistics etc.
     - polygons to instance masks and vise-versa
     - apply a custom colormap for mask annotations
     - rename or remove dataset labels
+  - Dataset splitting
```

Comment:
Maybe, it is worth adding a descriptive example in README.

Reply:
Okay, I'll add more description.